picture-game/src/components/Header.jsx

```diff
@@ -2,12 +2,13 @@ import React from 'react'
 import styled from 'styled-components'
 
 const Title = styled.div`
-    color:red;
-    font-size:14px;
+    color:white;
+    font-size:30px;
+    font-weight: bold;
```

Comment:
```suggestion
const Title = styled.h1`
    color:white;
    font-size:30px;
    font-weight: bold;
```
이라고 여기서도 시멘틱태그 쓸수이써용!